geometry_msgs/Pose pos
geometry_msgs/Twist vel
float32 hdg
